frontend/public/components/storage/attach-storage.tsx

```diff
@@ -53,25 +54,26 @@ const AttachStorage: React.FC<AttachStorageFormProps> = (props) => {
     setActiveProvider(id as string);
   };
 
-  const title = t('public~Add Storage');
   return !kindObj && kindsInFlight ? (
     <LoadingBox />
   ) : (
     <div className="co-m-pane__body">
       <Helmet>
-        <title>{title}</title>
+        <title>{t('public~Add Storage')}</title>
       </Helmet>
       <div className="co-storage-heading__wrapper">
-        <h1 className="co-m-pane__heading">{title} </h1>
-        <div className="co-m-pane__explanation co-storage-heading__subtitle">
-          to{' '}
-          <ResourceLink
-            inline
-            kind={props?.kindObj?.kind}
-            name={match.params.name}
-            namespace={match.params.ns}
-          />
-        </div>
+        <Trans t={t} ns="public">
+          <h1 className="co-m-pane__heading">Add Storage </h1>
+          <div className="co-m-pane__explanation co-storage-heading__subtitle">
+            to{' '}
+            <ResourceLink
+              inline
+              kind={props?.kindObj?.kind}
+              name={match.params.name}
+              namespace={match.params.ns}
+            />
+          </div>
```

Comment:
It's probably better to move the space outside the `h1` element for the translators, although I'm not sure if that will change the margin

```suggestion
          <h1 className="co-m-pane__heading">Add Storage</h1>
          <div className="co-m-pane__explanation co-storage-heading__subtitle">
            {' '}to{' '}
            <ResourceLink
              inline
              kind={props?.kindObj?.kind}
              name={match.params.name}
              namespace={match.params.ns}
            />
          </div>
```

Reply:
All set!